src/nixpacks/builder/docker/dockerfile_generation.rs

```diff
@@ -354,22 +353,27 @@ impl DockerfileGenerator for StartPhase {
                   {user_str}
                   {start_cmd}
                 ",
-                run_image=run_image,
-                APP_DIR=APP_DIR,
-                copy_cmds=copy_cmds.join("\n"),
-                user_str=user_str,
-                start_cmd=start_cmd,}
-            }
-            None => {
-                formatdoc! {"
+            run_image=run_image,
+            APP_DIR=APP_DIR,
+            copy_cmds=copy_cmds.join("\n"),
+            user_str=user_str,
+            start_cmd=start_cmd,}
+        } else {
+            // Copy over app files if provided, otherwise copy all files
+            let copy_cmds = match &self.only_include_files {
```

Comment:
START PHASE: if only_include_files is specified, then use it. Otherwise default to COPY . /app

Reply:
I believe `only_include_files` was only used when the runtime image was different from the build image. So this makes the uses the same 👍